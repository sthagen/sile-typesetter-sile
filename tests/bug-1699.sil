\begin[class=plain,papersize=a7]{document}
\use[module=packages.retrograde,target=v0.15.0]
\nofolios
% Both "bar" entries should have parindent applied
\noindent
\em{foo}

bar

\noindent
foo

bar
\end{document}
