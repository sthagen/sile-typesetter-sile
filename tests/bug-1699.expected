Set paper size 	209.7637818	297.6377985
Begin page
Mx 	10.4882
My 	22.3038
Set font 	Gentium Plus;10;400;Italic;normal;;;LTR;
T	73 82 82 w=11.1670	(foo)
Mx 	30.4882
My 	34.3038
Set font 	Gentium Plus;10;400;;normal;;;LTR;
T	69 68 85 w=13.6279	(bar)
Mx 	10.4882
My 	46.3038
T	73 82 82 w=13.1689	(foo)
Mx 	30.4882
My 	58.3038
T	69 68 85 w=13.6279	(bar)
End page
Finish
